### ISSUE TITLE
[v5] Resume alternative for v5

### ISSUE THREAD

**Issue by destruc7i0n** [rt2zz/redux-persist#561]
With the `persistor.resume()` function being removed in v5 and not really documented, is the current replacement `persistor.persist()`? Was a little confused when I saw that it was removed without any documentation.

**Comment by rt2zz** [comment]
correct, `persistor.persist()` is the new resume - persist method was required for codeplitting, and since the implementation would be identical we decided to simply omit resume. Agreed this should be in the docs.
